src/clr-angular/data/datagrid/datagrid.html

```diff
@@ -31,12 +31,12 @@
                 </div>
                 <!-- header for single row action; only displayType if we have at least one actionable row in datagrid -->
                 <div role="columnheader" class="datagrid-column datagrid-row-actions datagrid-fixed-column"
-                     *ngIf="rowActionService.hasActionableRow">
+                     *ngIf="rowActionService.hasActionableRow" [attr.aria-label]="commonStrings.select">
```

Comment:
Could you add a corresponding test for this change? Also, is it even a right pattern to place `aria-label` on a non-interactive element such as this `div` element here?

Reply:
There is no existing text that could be used for labeling so that's why I added `aria-label`. 

From the spec of `aria-label`, I don't see any issue adding it to any type of element.

@chlane 👷 


About the tests, give me a sec;)